src/tests/Microsoft.PowerFx.Interpreter.Tests/CheckResultExpectedReturnTests.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Linq;
+using System.Reflection.Metadata;
+using System.Threading;
+using Microsoft.PowerFx.Core.Tests;
+using Microsoft.PowerFx.Functions;
+using Microsoft.PowerFx.Types;
+using Xunit;
+
+namespace Microsoft.PowerFx.Interpreter.Tests
+{
+    public class CheckResultExpectedReturnTests : PowerFxTest
```

Comment:
>CheckResultExpectedReturnTests [](http://example.com/codeflow?start=17&length=30)

Since CheckResult is on MS.PFx. **Core**, can this test be on MS.PFx. **Core** .Tests?